### DISCUSSION TITLE
Welcome to the machine learning course for image processing

### DISCUSSION THREAD

**Discussion by astaaudzi** [fishsizeproject/Course-MLforImageProcessing#1] [General]
Hello everyone and welcome to the course. Here you can introduce yourself and tell others about your work. I am Asta Audzijonyte and I work at Nature Research Centre (Lithuania) and University of Tasmania (Australia). I am interested in various fish related questions, and am very excited about machine learning tools to improve data collection in fisheries.

**Comment by CatarinaNSSilva** [comment]
Hi everyone! I am Catarina Silva, currently working at Nature Research Centre (Lithuania), remotely from Portugal. My research has mostly focused on understanding the processes related to population dynamics with strong application to fisheries research. I am very interested in using machine learning tools to assess fish populations status.

Asta and I are currently working on the course materials and since this is the first time we are running this course, we are very interested in hearing any feedback you might have so we can improve this course for the research community in the future.

Looking forward to meet you all and share experiences and knowledge!

Catarina

**Comment by Zemmos** [comment]
Hello everybody!
My name is Igor Talijančić and I am a postdoctoral researcher at the Institute of Oceanography and Fisheries in Split, Croatia (http://www.izor.hr/web/guest). I work in the aquaculture laboratory and I am currently studying the environmental impact of aquaculture on local marine populations. In particular, I am studying the interactions between wild and farmed gilthead sea bream (Sparus aurata) and sea bass (Dicentrarchus labrax) and their phenotypic plasticity, as well as the FAD (fish aggregating devide) effects of tuna farming on populations of fish species. I use morphological tools such as body shape, scales, and otoliths to distinguish specimens by origin (wild, farmed, and farm-associated ), and in conjunction with geometric morphometrics, I quantify body shape forms in wild, farmed and farm-associated populations. I am very interested in learning and applying machine learning tools in aquaculture to improve data collection and assess which traits contribute most in distinguishing the origin of fish species.
I am interested in the use of image processing and pattern recognition technologies as they are proving to be particularly useful in distinguishing external appearance characteristics of fish, as they increase the performance and applicability of morphometric methods in identifying escapes in wild populations or fisheries landings by simultaneously recording multiple of body and scale shape. Through this course, I hope to gain insights and ideas for developing a machine learning algorithm that automatically identifies the origin of fish from digital images, as this would be of great importance to my research not only from a scientific perspective, but also from a commercial and non-commercial perspective, as it will (i) provide managers with a user-friendly tool to quickly and objectively identify the origin of fish, which is necessary to create a model proposal for recapturing escaped fish in the wild, and (ii) provide stakeholders with better insight into the origin of fish, thereby improving the traceability of the product.

Looking forward to meet you all soon!

Sincerely,
Igor

**Reply by astaaudzi** [reply]
Hi Igor, it is interesting to hear about your work. I think (hope) the course should give you some starting points on developing your own algorithms. Identifying fishes and different phenotypes should be doable if images and clear and you have enough of them. 
See you soon

**Reply by CatarinaNSSilva** [reply]
Hi Igor, your works sounds very interesting. I hope this course gives you the tools you need to develop your own model. Maybe start with a few images (say 50) for 2 classes (wild and farmed fish) and train your own model using the scripts if the course?

**Comment by fakhrurrozi-github** [comment]
Hello Everyone, my name is Ozi. 
I currently staff in Indonesian Coral Reef Foundation (TERANGI) from Indonesia. It is an amazing opportunity for me to be able to join this course. This is new for me learn about fishes.  I hope can full involved until finished !

Can't wait to see You all soon

**Reply by astaaudzi** [reply]
Hi Ozi, it is great that you are joining the course from Indonesia and I hope what you learn might become useful for your work at the Coral Reef Foundation

**Reply by CatarinaNSSilva** [reply]
Hi Ozi, hope you find the course interesting! See you soon

**Comment by vunguyen1907** [comment]
Hi Everyone. I am Vu Nguyen, a postdoctoral researcher at James Cook University, Singapore. My background is in quantitative genetics and genomics, targeting aquaculture genetic improvement breeding programs. Recently, I am interested in using machine & deep learning but my knowledge about it is shallow or almost nothing. In my current projects, we will record the fish (video) and try to measure the range of measurements such as weight, length, and tag,... so I think joining this course may improve my understanding a lot. I think machine learning and computer vision are such useful ways to cope with this.
Regards,
Vu

**Reply by CatarinaNSSilva** [reply]
Hi Vu, I hope this course will give you an overview of the applications of machine learning and computer vision and that it is useful for your work at JCU. See you soon!

**Comment by aalvarezellacuria** [comment]
Hi, I am Amaya Alvarez from IMEDEA institute at the Mediterranean Sea, SPAIN.
Is any of you having troubles to get into the zoom meeting?

**Reply by vunguyen1907** [reply]
Hi Amaya. I think it will start soon, 40 minutes from now.

**Reply by astaaudzi** [reply]
Hi Amaya, the zoom meeting has not started yet. It will start in 40 minutes

**Comment by ErleLan** [comment]
Hi, I am Erlend L. Langhelle from Demersal Fish Group in IMR in Bergen, Norway. I am a marine research engineer, and I specialize in reading cod (_Gadus morhua_) otoliths (age determination). I was recently part of a project group who used machine learning for age determination, and I want to get at deeper understanding of the mechanisms

**Reply by CatarinaNSSilva** [reply]
Hi Erlend, I hope you get a better understanding of the process in this course. I think ML application to otoliths age determination is a very promising area, it would be a great tool to have for data processing automation.

**Comment by IngridMBruvold** [comment]
Hello! I'm Ingrid Bruvold, and I work as a technician in the Deepwater and Cartilaginous Fish research group at the Institute of Marine Research (IMR) located in the north of Norway. My interest is largely in combining genetic data with morphology in highly similar species, mostly in redfish (_Sebastes_ spp.). Until now I've only worked with traditional morphometric methods, and I'm excited to learn more about the possibilities and utilization of machine learning for image based classification.

**Reply by CatarinaNSSilva** [reply]
Hi Ingrid, It would be really interesting to see if the model would be able to identify these redfish species correctly (and further compare model results with the morphometric analyses). Are you planning to train a ML model with your own images in the course?

**Reply by IngridMBruvold** [reply]
Hi Catarina! Yes, I have a few pictures of redfish to train the model with for this course. :)
